### PR TITLE
Added dark scheme tiles support

### PR DESCRIPTION
**Pull request by Petrprogs**
1. Added dark scheme tiles support.
2. Added corresponding fields in settings.
3. Added ranslations for settings (!!! **Review all langs, please** !!!)

### CODE REVIEW

File: src/mahomaps/screens/SettingsScreen.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public class SettingsScreen extends Form implements CommandListener {
 	private ChoiceGroup download = new ChoiceGroup(MahoMapsApp.text[54], Choice.MULTIPLE,
 			new String[] { MahoMapsApp.text[17] }, null);
 	private ChoiceGroup map = new ChoiceGroup(MahoMapsApp.text[153], Choice.EXCLUSIVE,
-			new String[] { MahoMapsApp.text[55], MahoMapsApp.text[154], MahoMapsApp.text[155] }, null);
+			new String[] { MahoMapsApp.text[55],  MahoMapsApp.text[167], MahoMapsApp.text[154], MahoMapsApp.text[155] }, null);
```
Comment (shinovon) [inline src/mahomaps/screens/SettingsScreen.java:32]: Map settings from previous versions will not be loaded correctly after that change, e. g satellite will be changed to dark, so I think there should be separate setting for scheme color

**Comment by shinovon** [comment]
plus I will have to do more presses on hotkey if I would like to change scheme to hybrid

File: src/mahomaps/screens/SettingsScreen.java (hunk 1 above)
Comment (Petrprogs) [reply src/mahomaps/screens/SettingsScreen.java:32]: I'm a bit dumb on this, so I probably can't make a separate parameter (yet) ;)

**Comment by Feodor0090** [comment]
I prefer to think that ordering must be locked. So, dark scheme must be the 4th scheme.

We 292% need selection menu here. Will implement right here.

**Comment by Feodor0090** [comment]
изображение

@shinovon Какая и где настройка отвечает за ширину "файла" при переносах по шифт+ктрл+ф? Нам надо всем поставить её в одно значение, но для начала надо вспомнить где она вообще

**Comment by shinovon** [comment]
> изображение
> 
> @shinovon Какая и где настройка отвечает за ширину "файла" при переносах по шифт+ктрл+ф? Нам надо всем поставить её в одно значение, но для начала надо вспомнить где она вообще

image

**Review by Feodor0090 — APPROVED** [review]
Tested to work

изображение

**Comment by Feodor0090** [comment]
This thing has no meaning, but I'm still interested in randomly changed access rights.
изображение
This happened only on files touched by @Petrprogs and this is first time I noticed this. Any details?

**Comment by Petrprogs** [comment]
> 

What's access rights?

**Comment by Petrprogs** [comment]
> This thing has no meaning, but I'm still interested in randomly changed access rights. изображение This happened only on files touched by @Petrprogs and this is first time I noticed this. Any details?



> This thing has no meaning, but I'm still interested in randomly changed access rights. изображение This happened only on files touched by @Petrprogs and this is first time I noticed this. Any details?

What's access rights?

**Comment by Feodor0090** [comment]
> > This thing has no meaning, but I'm still interested in randomly changed access rights. изображение This happened only on files touched by @Petrprogs and this is first time I noticed this. Any details?
> 
> What's access rights?

https://wiki.archlinux.org/title/File_permissions_and_attributes

I'm too rhuki iz zhopy to fix this so let's leave as is

**Comment by veselcraft** [comment]
@Feodor0090 ебать

**Comment by Feodor0090** [comment]
@veselcraft Кого?

**Pull request merged**